src/screens/SendSheet.js

```diff
@@ -283,30 +292,37 @@ export default function SendSheet(props) {
     // belongs to
     if (prevNetwork !== currentNetwork) {
```

Comment:
keeping in mind my note about not needing `transactionNetwork`, but separately: this if statement would not run if just `transactionNetwork` changes